package.json

```diff
@@ -61,6 +61,7 @@
     "cache-manager": "^5.4.0",
     "class-transformer": "^0.5.1",
     "class-validator": "^0.14.1",
+    "cron": "^3.1.7",
```

Comment:
There is a '@nestjs/schedule' [module ](https://docs.nestjs.com/techniques/task-scheduling)which does the same thing (probably uses cron under the hood), so we could consider using this instead of cron directly, just to make the most of using nestjs